quarkchain/cluster/master.py

```diff
@@ -1016,6 +1017,11 @@ def handle_new_root_block_header(self, header, peer):
         """ Add root block locally and broadcast root block to all shards and .
         All update root block should be done in serial to avoid inconsistent global root block state.
         """
+        block_hash = r_block.header.get_hash()
+        committed = self.root_state.is_committed_by_hash(block_hash)
```

Comment:
=> if self.root_state.is_committed_by_hash(...)  so that we could save one variable?